### PR TITLE
[ADD] New module to generate account move line for tax with amount zero

### PR DESCRIPTION
**Pull request by alessandrocamilli**

### CODE REVIEW

File: account_invoice_base_tax_zero/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+{
+    'name': 'Account Invoice Base Tax Zero',
+    'summary': 'Generate account move lines including tax lines with amount 0',
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'category': 'Accounting',
+    'author': 'Openforce,Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/account-invoicing/',
+    'depends': ['account'],
+    'data': [],
```
Comment (nikul-serpentcs) [inline account_invoice_base_tax_zero/__manifest__.py]: @alessandrocamilli IMO if not any file thne remove it

**Review by nikul-serpentcs — COMMENTED** [review]
Minor Change

**Review by eLBati — CHANGES_REQUESTED** [review]
Check travis plis

**Comment by alessandrocamilli** [comment]
@eLBati travis is ok now

File: account_invoice_base_tax_zero/__manifest__.py (hunk 1 above)
Comment (alessandrocamilli) [reply account_invoice_base_tax_zero/__manifest__.py]: line 'data' removed

**Comment by eLBati** [comment]
Please add at least DESCRIPTION.rst
see https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: account_invoice_base_tax_zero/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+For existing invoice, it needed to re-validate them.
```
Comment (eLBati) [inline account_invoice_base_tax_zero/readme/USAGE.rst]: `it is necessary`?

File: account_invoice_base_tax_zero/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,9 @@
+This module in case of invoice with amount zero, generates in account move
```
Comment (eLBati) [inline account_invoice_base_tax_zero/readme/DESCRIPTION.rst]: `This module, in case`

Comment (eLBati) [reply account_invoice_base_tax_zero/readme/DESCRIPTION.rst]: `generates in`?

File: account_invoice_base_tax_zero/readme/DESCRIPTION.rst [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+lines for tax and partner.
+
+This module is needed to vat reports where is important to see vat moves,
+also when are zero.
+
+The module account_tax_balance, when a move is zero, sets always the account move type
+how refund. So this module in case of invoice with amount zero, provides to set
```
Comment (eLBati) [inline account_invoice_base_tax_zero/readme/DESCRIPTION.rst]: `how refund` > `as refund`?

File: account_invoice_base_tax_zero/readme/DESCRIPTION.rst [hunk 5]
```diff
@@ -0,0 +1,9 @@
+This module in case of invoice with amount zero, generates in account move
+lines for tax and partner.
+
+This module is needed to vat reports where is important to see vat moves,
```
Comment (eLBati) [inline account_invoice_base_tax_zero/readme/DESCRIPTION.rst]: `where it is important`

File: account_invoice_base_tax_zero/readme/DESCRIPTION.rst [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+also when are zero.
```
Comment (eLBati) [inline account_invoice_base_tax_zero/readme/DESCRIPTION.rst]: `when they are zero`

**Pull request closed without merging**